landr.conf.js

```diff
@@ -0,0 +1,62 @@
+const prepAssets = require('./www/prepAssets')
+
+module.exports = {
+  theme: 'landr-theme-basic',
+  middleware: (store, action, next) => {
```

Comment:
Can we replace this with a function that simply takes a filename, and returns an object? For example:

```js
parseReleaseFile: (filename) => {
    // Some magic here...

    return {
        os: 'darwin',
        type: 'cli',
        ...
    }
}
```